File: eternal_return/src/main/java/org/EternalReturn/EREntity/ERDummy.java
package org.EternalReturn.EREntity;

import org.EternalReturn.EREntity.GlobalMonobehav.InfinityHealing;
import org.EternalReturn.System.PluginInstance;
import org.bukkit.entity.Entity;
import org.bukkit.entity.LivingEntity;

public class ERDummy extends EREntity{

    public ERDummy(Entity entity){
        super(PluginInstance.getEREngine().createCylinder(
                PluginInstance.getEREngine().createInfStrightLine(0,1,0,
                        entity.getLocation().getX(),
                        entity.getLocation().getY(),
                        entity.getLocation().getZ()),
                1,3));
        this.setEntity(entity);
        ((LivingEntity)entity).setAI(false);

//        registerMonobehaviour(new InfinityHealing());
    }

}

File: eternal_return/src/main/kotlin/org/EternalReturn/EREntity/ERDummy.kt
package org.eternalreturn.erentity

import org.eternalreturn.system.PluginInstance
import org.bukkit.entity.Entity
import org.bukkit.entity.Player
import org.eternalreturn.erplayer.ERPlayer
import org.eternalreturn.system.EREngine
import org.eternalreturn.util.dpengine.DPEngine
import org.eternalreturn.util.dpengine.geometry.Collider

class ERDummy(engine : EREngine, entity: Entity, collider: Collider) : EREntity(engine, entity, 0.5, 1.0, 0.5, 0.0, 1.0 ,0.0) {
    init {
        //(entity as LivingEntity).setAI(false)
        //registerMonobehaviour(new InfinityHealing());
    }
}
